sci-libs/hipBLAS/hipBLAS-4.3.0.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+DESCRIPTION="ROCm BLAS marshalling library"
+HOMEPAGE="https://github.com/ROCmSoftwarePlatform/hipBLAS"
+SRC_URI="https://github.com/ROCmSoftwarePlatform/hipBLAS/archive/rocm-${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+KEYWORDS="~amd64"
+SLOT="0/$(ver_cut 1-2)"
+
+RDEPEND="dev-util/hip:${SLOT}
+	sci-libs/rocBLAS:${SLOT}
+	sci-libs/rocSOLVER:${SLOT}"
+DEPEND="${RDPEND}"
+BDEPEND=""
+
+S="${WORKDIR}/hipBLAS-rocm-${PV}"
+
+src_prepare() {
+	sed -e "s:<INSTALL_INTERFACE\:include:<INSTALL_INTERFACE\:include/hipblas/:" -i library/src/CMakeLists.txt || die
+	sed -e "/PREFIX hipblas/d" -i library/src/CMakeLists.txt || die
+	sed -e "/rocm_install_symlink_subdir( hipblas )/d" -i library/src/CMakeLists.txt || die
+	sed -e "s:get_target_property( HIPHCC_LOCATION hip\:\:hip_hcc IMPORTED_LOCATION_RELEASE ):get_target_property( HIPHCC_LOCATION hip\:\:hip_hcc IMPORTED_LOCATION_GENTOO ):" -i library/src/CMakeLists.txt
```

Comment:
This line is too long.  Try shorten it.

Can it be
```
/HIPCC_LOCATION/s/IMPORTED_LOCATION_RELEASE/IMPORTED_LOCATION_GENTOO/
```
?

Reply:
I removed this line because it's does not modified anything.